iothub_client/tests/blob_ut/blob_ut.c

```diff
@@ -119,7 +119,7 @@ TEST_DEFINE_ENUM_TYPE(BLOB_RESULT, BLOB_RESULT_VALUES);
 #define X_MS_BLOB_TYPE "x-ms-blob-type"
 #define BLOCK_BLOB "BlockBlob"
 
-MU_DEFINE_ENUM_STRINGS(UMOCK_C_ERROR_CODE, UMOCK_C_ERROR_CODE_VALUES)
+MU_DEFINE_ENUM_STRINGS_WITHOUT_INVALID(UMOCK_C_ERROR_CODE, UMOCK_C_ERROR_CODE_VALUES)
```

Comment:
>MU_DEFINE_ENUM_STRINGS_WITHOUT_INVALID [](start = 0, length = 38)

I don't believe this is correct, I think the UMOCK needs to be MU_DEFINE_ENUM_STRINGS(